src/pykeen/evaluation/rank_based_evaluator.py

```diff
@@ -383,7 +392,7 @@ def process_head_scores_(
         self._update_ranks_(true_scores=true_scores, all_scores=scores, side=LABEL_HEAD, hrt_batch=hrt_batch)
 
     def _get_ranks(self, side: ExtendedTarget, rank_type: ExtendedRankType) -> np.ndarray:
-        if side == SIDE_BOTH:
+        if side == SIDE_BOTH:  # TODO should this include LABEL_RELATION?
```

Comment:
@mberr I changed the implementations of the stuff that should wait for future PRs to just be stub not implemented error raises, but this is still a major question for this PR - should `SIDE_BOTH` correspond to just head+tail or is it all three?

Reply:
The default should remain to evaluate on head & tail, since this is the default in literature.